spec/spec_helper_acceptance.rb

```diff
@@ -1,44 +1,29 @@
-require 'beaker-rspec/spec_helper'
-require 'beaker-rspec/helpers/serverspec'
+require 'beaker-rspec'
+# Helper does not yet support Puppet 5
+#require 'beaker/puppet_install_helper'
+require 'beaker/module_install_helper'
 
-unless ENV['RS_provision'] == 'no'
-  hosts.each do |host|
-    if host.is_pe?
-      install_pe
-    else
-      install_puppet
-      on host, "mkdir -p #{host['distmoduledir']}"
-    end
-  end
-end
+# Helper does not yet support Puppet 5
+#install_puppetlabs_release_repo_on(hosts, 'puppet5')
+install_puppet_agent_on(hosts, :puppet_collection => 'puppet5', :puppet_agent_version => ENV['PUPPET_INSTALL_VERSION'])
+#run_puppet_install_helper
+install_module_on(hosts)
+install_module_dependencies_on(hosts)
 
 UNSUPPORTED_PLATFORMS = ['Suse','windows','AIX','Solaris']
 
 RSpec.configure do |c|
-  # Project root
-  proj_root = File.expand_path(File.join(File.dirname(__FILE__), '..'))
-
   # Readable test descriptions
   c.formatter = :documentation
 
   # Configure all nodes in nodeset
   c.before :suite do
     # Install module and dependencies
     hosts.each do |host|
-      copy_module_to(host, :source => proj_root, :module_name => 'sensu')
-
-      if fact('osfamily') == 'Debian' and fact('operatingsystemmajrelease') == '12.04'
-        # RubyGems missing on some Ubuntu 12 boxes
-        # Otherwise you'lll get a load of 'Provider gem is not functional on this host'
-        shell('apt-get install rubygems -y')
-      end
       if fact('osfamily') == 'RedHat'
         # CentOS has epel-release package in Extras, enabled by default
         shell('yum -y install epel-release')
-        shell('yum -y install erlang')
       end
-      on host, puppet('module', 'install', 'puppetlabs-stdlib'), { :acceptable_exit_codes => [0,1] }
```

Comment:
why does stdlib go away here?

Reply:
Pulled in from `metadata.json` by `install_module_dependencies_on(hosts)` at the top of file.  Removed to avoid having to update some dependencies in two places.